quirks/shared-credentials.json

```diff
@@ -135,6 +135,17 @@
             "diversalertnetwork.org"
         ]
     },
+    {
+        "from": [
+            "digiromania.ro",
```

Comment:
I don't see digiromania.ro redirecting. When I load it, it shows a page at that domain!

Reply:
Huh 🤔 Even in incognito?

```bash
$ curl --include digiromania.ro
HTTP/1.1 301 Moved Permanently
Date: Sat, 28 Sep 2024 07:24:40 GMT
Content-Type: text/html
Content-Length: 162
Connection: keep-alive
Location: https://www.digi.ro/
Server: RDS-WebServer v2

<html>
<head><title>301 Moved Permanently</title></head>
<body>
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx</center>
</body>
</html>
```